js/widgets/forms/button.backcompat.js

```diff
@@ -23,8 +23,8 @@
 			"jquery",
 			"../../core",
 			"../../widget",
-			"../widget.backcompat",
-			"./button" ], factory );
+			"./button",
+			"../widget.backcompat" ], factory );
```

Comment:
yeah i think this is just from debugging i accidently switched it at some point will revert